ams/server/src/main/java/com/netease/arctic/server/DefaultOptimizingService.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server;
+
+import com.google.common.base.Preconditions;
+import com.netease.arctic.ams.api.OptimizerRegisterInfo;
+import com.netease.arctic.ams.api.OptimizingService;
+import com.netease.arctic.ams.api.OptimizingTask;
+import com.netease.arctic.ams.api.OptimizingTaskId;
+import com.netease.arctic.ams.api.OptimizingTaskResult;
+import com.netease.arctic.ams.api.resource.ResourceGroup;
+import com.netease.arctic.server.exception.ObjectNotExistsException;
+import com.netease.arctic.server.exception.PluginRetryAuthException;
+import com.netease.arctic.server.optimizing.OptimizingQueue;
+import com.netease.arctic.server.optimizing.OptimizingStatus;
+import com.netease.arctic.server.persistence.mapper.OptimizerMapper;
+import com.netease.arctic.server.persistence.mapper.ResourceMapper;
+import com.netease.arctic.server.resource.DefaultResourceManager;
+import com.netease.arctic.server.resource.OptimizerInstance;
+import com.netease.arctic.server.resource.OptimizerManager;
+import com.netease.arctic.server.table.DefaultTableService;
+import com.netease.arctic.server.table.RuntimeHandlerChain;
+import com.netease.arctic.server.table.TableConfiguration;
+import com.netease.arctic.server.table.TableManager;
+import com.netease.arctic.server.table.TableRuntime;
+import com.netease.arctic.server.table.TableRuntimeMeta;
+import com.netease.arctic.table.ArcticTable;
+import org.apache.commons.collections.CollectionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+/**
+ * OptimizerManagementService is implementing the OptimizerManager Thrift service, which manages the optimization tasks
```

Comment:
`OptimizerManagementService` is a existing class?

Reply:
I will fix it.